apps/website-new/docs/zh/guide/basic/chrome-devtool.mdx

```diff
@@ -52,7 +52,7 @@
 
 ## 如何将本地开发的模块代理到线上
 
-- 首先需要在本地启动 Vmok 生产者
+- 首先需要在本地启动生产者
```

Comment:
The English version also needs to be modified, the path `apps/website-new/docs/en/guide/basic/chrome-devtool.mdx`